sql/moz-fx-data-shared-prod/firefox_accounts_derived/nonprod_fxa_server_events_v1/metadata.yaml

```diff
@@ -0,0 +1,37 @@
+---
+friendly_name: FxA Server Events (nonprod)
+description: |
+  FxA server events extracted from accounts server stdout logs.
+
+  This new table is the direct result of FxA migration from AWS to GCP
+  (see: OPST-296 for more context).
+
+  Effective 2023-05-26 the events from the following servers land in this table
+  (existing data will remain in those tables):
+  - auth    (previous table: `nonprod_fxa_auth_events_v1`)
+  - content (previous table: `nonprod_fxa_content_events_v1`)
+
+  Payment server events will continue landing inside: `nonprod_fxa_stdout_events_v1`
+
+owners:
+  - kik@mozilla.com
+labels:
+  application: fxa
+  incremental: true
+  schedule: daily
+scheduling:
+  dag_name: bqetl_fxa_events
+  arguments: ['--schema_update_option=ALLOW_FIELD_ADDITION']  # after initing the table with one column this should just add all the other fields automatically
+bigquery:
+  time_partitioning:
+    type: day
+    field: timestamp
+    require_partition_filter: true
+  # cannot set clustering via this config file as no schema.yaml
+  # file due to how volatile the table schema can be.
+  # To mitigate this we create the table using the init.sql file
+  # which creates an empty table with the following fields and configuration:
+  # - timestamp TIMESTAMP - partition field
+  # - fxa_log STRING - cluster field
+  # the argument ALLOW_FIELD_ADDITION takes care of adding all the missing field
+  # as part of the ETL execution.
```

Comment:
we need to add this to dry-run skip anyways due to the source data being in a project to which the CI does not have access to.

Reply:
Copying what I wrote in [Slack](https://mozilla.slack.com/archives/C01E8GDG80N/p1686756243631509?thread_ts=1686596901.565209&cid=C01E8GDG80N) here for posterity/continued discussion:

Adding it to dry-run skip doesn't solve the CI problem, which has to do with changes to ETLs/views immediately downstream of such ETLs.  When CI tries to deploy all the things the changed ETL/view is referencing to staging it can't do that for any un-dry-runnable ETLs referenced unless they have a `schema.yaml` file.

For example, the `fxa_all_events` view selects from un-dry-runnable ETLs, so for any changes to `fxa_all_events` to successfully pass CI in order to get deployed the FxA events ETLs it references must have `schema.yaml` files, otherwise CI fails to deploy them to staging in order to run the required `validate-metadata` CI check (as I found in #3926).

Reply:
ok, I will add the remainder of the schema.yaml files. I will pull the nonprod ones from your PR into this one.

Reply:
schema files added. I consider this conversation resolved, please let me know if there's anything I missed.